### ISSUE TITLE
Trennstrich immer setzen nach einem ab mit corresp "epilog"

### ISSUE THREAD

**Issue by laMischke** [cceh/capitularia#50]
@Capitularia-sk @domtru @skheinen 

Wenn das @corresp eines ab "_epilog" enthält, soll nach dem `ab` immer ein Trennstrich folgen (außer, es gibt schon einen, weil der ab mit "epilog" ein `ab text` ist, auf den ein weiterer `ab text` folgt; also keine doppelten Trennstriche!).

**Comment by laMischke** [comment]
@Capitularia-sk @domtru @skheinen 

Ergänzung: Bitte auch nach einem ab meta-text mit einem @corresp, das "explicit" enthält, einen Trennstrich ergänzen!

**Comment by laMischke** [comment]
@Capitularia-sk @domtru @skheinen 

Uns fällt gerade auf, dass das Trennstrichsetzen im zweiten oben geschilderten Fall nicht immer so funktioniert, wie wir uns das gedacht haben:
- Der corresp-Wert kann auch "explicit" alleine enthalten (bei unserem Test hat das Trennstrichsetzen funktioniert, wenn wir den Wert "_explicit" genannt haben, aber nicht, wenn der Unterstrich fehlte);
- und noch eine Ergänzung: die corresp-Werte (mit) "explicit" bitte **nicht** auslesen (Anzeige in eckigen Klammern vor dem Text), wie bei denen mit "inscriptio"!

**Comment by laMischke** [comment]
@Capitularia-sk 
Ein weiterer Sonderfall, bei dem diese Regel einen unerwünschten Effekt hat: In der Hs. Paris 4613 hat der Epilog zu BK 215 eine Kapitelzählung, die jetzt durch einen Trennstrich vom Text abgesetzt ist.
`<ab type="meta-text" rend="coloured" xml:id="paris-bn-lat-4613-97v-7"
            corresp="BK.215_epilog_inscriptio"><seg type="num" n="6">VI</seg> . <lb/>
         </ab>`
Kann man für solche Fälle eine Ausnahme formulieren, also z. B. Wenn das corresp "epilog" die Erweiterung "_inscriptio" hat: Keinen Trennstrich setzen?

**Comment by Capitularia-sk** [comment]
Möglicherweise wäre als Regel geeignet:
"Wenn auf einen /ab/ mit einem corresp, der nach einem slug (wie 'BK.215_') den Ausdruck 'epilog_inscriptio' enthält (mit Vorkehrungen, damit auch die möglichen 'epilog_1_inscriptio'-Fälle abgefangen werden), ein weiterer /ab/ folgt, der den gleichen slug und den gleichen Ausdruck wie oben, nur ohne '_inscriptio', enthält, dann sollen die beiden /ab/ nicht durch einen Trennstrich voneinander abgesetzt werden."

**Comment by MarcelloPerathoner** [comment]
Paris 4613 funktioniert mittlerweile. Aber ist es nicht so, daß "inscriptio" in keinem Fall vom darauffolgenden Text getrennt werden sollte?

**Comment by laMischke** [comment]
@Capitularia-sk 
Ja; aber warum wurde dann bei Paris 4613 trotzdem einer gesetzt?
